packages/backend-common/api-report.md

```diff
@@ -456,13 +461,68 @@ export { isChildPath };
 // @public
 export function isDatabaseConflictError(e: unknown): boolean;
 
+// @public
+export class KubernetesContainerRunner implements ContainerRunner {
+  constructor(options: KubernetesContainerRunnerOptions);
+  // (undocumented)
+  createJob(jobSpec: V1Job): Promise<any>;
+  // (undocumented)
+  getNamespace(kubeConfig: KubeConfig, namespace?: string): string;
+  // (undocumented)
+  handleError(err: any, errorCallback: (reason: any) => void): void;
+  // (undocumented)
+  runContainer(options: RunContainerOptions): Promise<void>;
```

Comment:
This should be the only method that should be made public right?
Would be nice to slim down the API surface of what we export :)